Task 2.0/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_2._0
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите радиус окружности");
+            var radius = double.Parse(Console.ReadLine());
+            var AreaOfSquare = 2 * Math.Pow(radius, 2);
```

Comment:
Квадрат лучше вычислять через произведение. Локальные переменные принято назвать с маленькой буквы.
```suggestion
            var areaOfSquare = 2 * radius * radius;
```